src/frontend/screens/Game/GamePage/index.tsx

```diff
@@ -140,6 +141,11 @@ export default observer(function GamePage(): JSX.Element | null {
 
   const storage: Storage = window.localStorage
 
+  const uncompressedSize = useEstimatedUncompressedSize(
+    platform,
+    gameInstallInfo?.manifest?.disk_size || 0
+  )
```

Comment:
maybe we should only guess the uncompressed size if `installSize !== downloadSize` i.e. if we actually have the value